tests/data/bundle.yaml

```diff
@@ -9,16 +9,16 @@ machines:
     series: focal
 services:
   containerd:
-    charm: cs:~containers/containerd
+    charm: containerd
     channel: edge
```

Comment:
i've been pushing these to `latest/edge` for when we begin to use the other tracks like `1.23/edge`.  I don't know if that's actually necessary -- but at least its more explicit